packages/css-library/src/utilities/border.scss

```diff
@@ -0,0 +1,51 @@
+@use 'uswds-core/src/styles/functions' as *;
+@use '../tokens/border' as *;
+@use '../tokens/color' as *;
+@use './settings' as *;
+
+$u-border-base: (
+  border: (
+    base: 'vads-border',
+    modifiers: (
+      noModifier: '',
+      'top': '-top',
+      'right': '-right',
+      'bottom': '-bottom',
+      'left': '-left',
+    ),
+    values: map-collect($tokens-border),
+    settings: $responsive,
+    property: 'border',
+    type: 'utility',
+  ),
+);
+
+$u-border-style: (
+  border-style: (
+    base: 'vads-border-style',
```

Comment:
One thing that we haven't explicitly talked about is naming patterns for this new library. With the color utilities I dropped the `color` from the name since a utility class like `vads-gray` is unambiguously for a color, and [shortened `background` to `bg`](https://github.com/department-of-veterans-affairs/component-library/blob/6bcfbcb543d06b3183c72365b72c79de9f642635/packages/css-library/src/utilities/background-color.scss#L8).

This will gives us `vads-border-style-dashed`, when `vads-border-dashed` can communicate the same thing with fewer characters, and thus a smaller stylesheet.

This can be applied to other classes - `vads-font-md` and `vads-font-2xl` seems like they can be interpreted as changing the `font-size` property even without explicitly mentioning size in the name. However, there are cases where removing that extra word makes the classes become ambiguous. Font style and font weight are examples of this.

With the current PR we would get `vads-font-style-normal` and `vads-font-weight-normal`. The font style would have additional `-italic` and `-bold` utilities, and the font weight style would have `bold` as well.

My question for us and @k80bowman is, do we want to document how we choose the names for these new utilities? Should we go for performance & usability when we can, or should we be as specific as possible no matter what?

Reply:
I would agree that performance and usability are our primary concerns with naming these utilities. I would also say yes, we should absolutely document how we choose the names, if for no other reason than we can point to it when people ask. I could see this actually being part of our [engineering best practices doc](https://vfs.atlassian.net/wiki/spaces/DST/pages/2176516116/Design+System+Engineering+Best+Practices).

Reply:
@bkjohnson That makes sense. I'll update utilities to use fewer characters without being ambiguous.